platform/android/export/export.cpp

```diff
@@ -744,6 +744,21 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return OK;
 	}
 
+	// Writes string p_data to a file at p_path, creating directories as necessary.
+	static Error store_string_at_path(const String &p_path, const String &p_data, int compression_method = Z_DEFLATED) {
```

Comment:
Function parameters should be prefixed with `p_` (or `r_` if they are "return" parameters that get mutated in the function).

```suggestion
	static Error store_string_at_path(const String &p_path, const String &p_data, int p_compression_method = Z_DEFLATED) {
```

Reply:
Thank you for the clarification, I will follow this from now on.